web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentList.php

```diff
@@ -9,9 +9,7 @@
 
 use DBCDK\CommunityServices\ApiException;
 use Drupal\Core\Block\BlockBase;
-use Drupal\Core\Logger\LoggerChannelFactory;
 use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
-use Drupal\dbcdk_community\Content\FlaggableContent;
```

Comment:
These are used in DocBlocks.

If we delete them here then we need to use fully qualified classnames there.